features/core_trad_register_client.feature

```diff
@@ -16,7 +16,7 @@ Feature: Register a client
     And I click on the css "a#clear-ssm"
     And I wait for the data update
     Then I should see a "System Status" text
-    And I should see a "Software Updates Available" text
+    And I should see a "Software Updates Available" or "System is up to date" text
```

Comment:
I understand that, but as this is a core feature, we should decide whether it should show one or other label depending on the actual status of the traditional client at this point.

Also notice that vendor channels (created by core_srv_sync_channels) are created and assigned after this step, in the latest core feature step. So it's not expected software updates unless you provide them using some custom repository (like virgo, andromeda, etc).